project.go

```diff
@@ -163,8 +163,16 @@ func (rs project) Run(ctx context.Context, inp, out chan Dataset) (err error) {
 	}()
 
 	// collect & join the output from all runners, in order
+	var resultLen int
```

Comment:
conceptually it's width, not length:)
```suggestion
	var resultWidth int
```